src/main/kotlin/eu/kevin/api/models/exception/KevinApiClientErrorException.kt

```diff
@@ -4,10 +4,12 @@ import kotlinx.serialization.Serializable
 import kotlinx.serialization.json.JsonElement
 
 class KevinApiClientErrorException internal constructor(
-    val response: ErrorResponse
+    val responseStatusCode: Int,
+    val responseBody: ClientError?
```

Comment:
name `responseBody` does not match `ClientError` naming. I would consider using `KevinApiClientErrorResponseBody` or something like this

Reply:
`ClientError` is already under the namespace `KevinApiErrorException`. Perhaps we should rename the parameter `responseBody` to something like `clientError`, since it is the only type of body that can exist.